packages/drivers/odsp-driver/src/odspDocumentStorageManager.ts

```diff
@@ -520,7 +574,7 @@ export class OdspDocumentStorageService implements IDocumentStorageService {
                 headers: Object.keys(headers).length !== 0 ? true : undefined,
                 sprequestguid: response.headers.get("sprequestguid"),
                 sprequestduration: TelemetryLogger.numberFromString(response.headers.get("sprequestduration")),
-                isOptionsCall,
+                isoptionscall: isOptionsCall,
```

Comment:
Capitalization of property name was lost

Reply:
Actually, it is consistent with the other property in this event, like all other properties are in small letters. So made consistent with other ones.